src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -262,7 +294,6 @@ export const ImageForm = ({
 									? t('warnings.image_deployed_to_docker')
 									: ''
 							}
-							onClick={() => setDownloadConfigOnly(false)}
```

Comment:
Removing this onClick means clicking this option now does nothing. So if I click the config option and the button switches to config, there is no (reasonable) way for me to switch back to OS

Reply:
thank you for noticing, this is a last minute change that does not have any sense